recipes/libgeoip/bld.bat

```diff
@@ -0,0 +1,6 @@
+SET GEOIPDATADIR=%LIBRARY_PREFIX%
```

Comment:
Change this to `SET GEOIPDATADIR=%LIBRARY_PREFIX%\share\GeoIP` to make it consistent with unixes.